src/pages/user/user.module.scss

```diff
@@ -0,0 +1,203 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```

Comment:
Remove this, poppins already exists; it is imported at the top level see that here https://github.com/castynet-studios/pms-fe/blob/main/src/index.scss#L1

Reply:
Done